whois-commons/src/test/java/net/ripe/db/whois/common/dao/jdbc/DatabaseHelper.java

```diff
@@ -249,9 +257,9 @@ private static DataSource createDataSource( final String databaseName) {
     }
 
     static void ensureLocalhost(final JdbcTemplate jdbcTemplate) {
-        final boolean isLocalhostOrRdonly = jdbcTemplate.execute(new ConnectionCallback<Boolean>() {
+        final boolean isLocalhostOrRdonly = Boolean.TRUE.equals(jdbcTemplate.execute(new ConnectionCallback<Boolean>() {
```

Comment:
Can execute() return `null` ? Better to fail on assigning to a boolean rather than default to `true`

Reply:
i would suppose that autoboxing would come into play then it's not so clear what would happen -- i just wanted to make it explicit. but yes, your suggestion is better still. i'll do that.

Reply:
added extra check